examples/jetpack_compose/compose-ui/MainActivity.kt

```diff
@@ -1,19 +1,21 @@
 package cm.ben.android.bazel.compose.example.ui
 
 import android.os.Bundle
+import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.platform.setContent
+import androidx.compose.ui.tooling.preview.Preview
 
 class MainActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContent {
-            Greeting("world")
-        }
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContent {
+      Greeting("world")
     }
+  }
 }
 
+@Preview
```

Comment:
Does this actually work with Android Studio and Bazel? I never got the previews to work.

Reply:
https://github.com/bazelbuild/intellij/issues/2765#issuecomment-973703195

Reply:
*sigh*

We've got some work to do before we have a complete working system.